crypt_test.go

```diff
@@ -220,6 +210,18 @@ func (ctb *cryptTestBase) addMockKeyslot(c *C, key []byte) {
 	ctb.mockKeyslotsCount++
 }
 
+func (ctb *cryptTestBase) newPrimaryKey(sz int) []byte {
```

Comment:
All primary keys are currently 32 bit so I wonder if maybe this should simply default to this? Or maybe that is the part that will be used in the future? OTOH there were just two PRs that changed the keysizes in the tests to match the real world so I wonder if maybe this should at least be a constant (defaultKeySize) or something?

Reply:
I ended up just removing the sz argument from this function.